styles/index.js

```diff
@@ -84,12 +84,9 @@ const themeExtensions = {
       css: {
         code: {
           fontWeight: '400',
-        },
-        'code::before': {
-          content: '',
-        },
-        'code::after': {
-          content: '',
+          '&::before, &::after': {
```

Comment:
Note that this might be written like this previously to override those parts of the tailwind typography, not 100% sure though. Please test before merging this code style before/after change!

Reply:
Changing this does introduce changes to the code styling:

![image](https://github.com/executablebooks/myst-theme/assets/913249/5c7a59ef-e21e-45b2-9141-4da5c8553d82)

H4 also already has styles! Updating.